### PR TITLE
[10.0] [mig] account_move_line_purchase_info

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Changes done on <code>account_move_line_purchase_info</code> migration to v10:
* Basic migration stuff (bump version, odoo tags...)
* Move method <code>_prepare_account_move_line</code>
* Fix test: Valuation method and stock input and output account now defined in product category.

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
@eficent @Darshan-serpent Can you please review?

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by AaronHForgeFlow** [comment]
It works fine when I have received the products from the PO. However if I don't receive the products and I create the invoice first the journal entries will have no journal items. I'l figure it out if this is because this module.

File: account_move_line_purchase_info/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2017 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
```
Comment (AaronHForgeFlow) [inline account_move_line_purchase_info/models/account_invoice.py]: odoo

File: account_move_line_purchase_info/tests/test_account_move_line_purchase_info.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+# © 2017 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests import common
+from openerp import fields
```
Comment (AaronHForgeFlow) [inline account_move_line_purchase_info/tests/test_account_move_line_purchase_info.py]: odoo

**Review by AaronHForgeFlow — COMMENTED** [review]
Pending testing commented issue

**Comment by LoisRForgeFlow** [comment]
Thanks @aheficent fixed now!

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Checked the issue commented. It is not due this module.

**Comment by MiquelRForgeFlow** [comment]
@lreficent what is the status of this PR?

**Comment by LoisRForgeFlow** [comment]
@mreficent I think is ready to review and merge!

**Pull request merged**